Source/Core/VideoCommon/VertexShaderGen.cpp

```diff
@@ -403,6 +519,86 @@ ShaderCode GenerateVertexShaderCode(APIType api_type, const ShaderHostConfig& ho
     out.Write("}}\n");
   }
 
+  if (uid_data->vs_expand == VSExpand::Line)
+  {
+    out.Write("// Line expansion\n"
```

Comment:
I didn't go through this in depth but I see really similar comments and code here.  Would we be able to move this (or parts of it?) to ShaderGenCommon in a function?

Reply:
Something like this?

Reply:
Beautiful.  Thank you!